packages/cw-orch-fns-derive/src/execute_fns.rs

```diff
@@ -128,7 +128,7 @@ pub fn execute_fns_derive(input: DeriveInput) -> TokenStream {
     let derived_trait = quote!(
         #[cfg(not(target_arch = "wasm32"))]
         /// Automatically derived trait that allows you to call the variants of the message directly without the need to construct the struct yourself.
-        pub trait #bname<Chain: ::cw_orch::prelude::TxHandler, #type_generics>: ::cw_orch::prelude::CwOrchExecute<Chain, ExecuteMsg = #entrypoint_msg_type #ty_generics> #where_clause {
+        pub trait #bname<Chain: ::cw_orch::prelude::TxHandler, CwOrchExecuteMsgType : From<#entrypoint_msg_type<#type_generics>>, #type_generics>: ::cw_orch::prelude::CwOrchExecute<Chain, ExecuteMsg = CwOrchExecuteMsgType> #where_clause {
```

Comment:
We should replace this `From` bound with an equivalent `Into` bound to loosen the type's requirement.